### DISCUSSION TITLE
♿Tab widget doesn't seem navigable with tab key anymore.

### DISCUSSION THREAD

**Discussion by websubst** [elementor/elementor#25363] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

For 508 compliance, I cannot find a way to navigate the Tab Widget with the tab key. It says here that it was resolved: https://github.com/elementor/elementor/issues/3576

### Describe the solution you'd like.

Do I have to press another key or do I need to add an attribute to navigate to a second tab?

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Daveden2** [comment]
According to [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/), we use the Tab key to focus on a tab title, then we use the Arrow keys to navigate between tab titles and finally the Enter key to activate a tab.

Isn't that how it works currently? I must admit that I haven't tested it properly in a while.

**Reply by websubst** [reply]
Hi @Daveden2 ,  I'm not able to replicate the problem and It is working now. Elementor improved accessibility in the past months. Thanks to you! 👍
